web/app/plugins/mitlib-pull-hours/mitlib-pull-hours.php

```diff
@@ -13,20 +13,30 @@
  * @link https://github.com/MITLibraries/mitlib-pull-hours
  */
 
-namespace Mitlib;
+namespace Mitlib\PullHours;
 
 // Don't call the file directly!
 if ( ! defined( 'ABSPATH' ) ) {
 	exit;
 }
 
-// Use the Composer autoloader to get all needed classes.
-require plugin_dir_path( __FILE__ ) . 'vendor/autoload.php';
+// Include the necesary classes.
+require( plugin_dir_path( __FILE__ ) . 'src/class-admin-widget.php' );
```

Comment:
Is it meaningful that we are using `require` here and not the `include_once` syntax used in other plugins?

Reply:
There is a difference between those statements, but I can't say that they were chosen correctly. Looking now, it should at least be `require_once` and not `require` (since these are classes rather than inline code, and cause problems if they're encountered multiple times).